Project/API/src/src/main/java/com/example/models/EmbededAssetModel.java

```diff
@@ -0,0 +1,28 @@
+package com.example.models;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.mongodb.lang.NonNull;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Document(collection = "Embeded_Asset")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class EmbededAssetModel {
+    private Number item_id;
```

Comment:
Ex: item_id should be not null
quntity must be greater than 0 (there validation we can use for this as we discussed like @Min(1))